tests/unit/client/test_mixin_client.py

```diff
@@ -339,9 +339,15 @@ def test_log_dataframe(project_client, test_dataframe):
     project = project_client
     df = test_dataframe
     test_df_name = "test_df"
-    dataframe = DataframeMixin.log_dataframe(project, df, name=test_df_name, tags=["x"])
-    DataframeMixin.log_dataframe(project, df, name="secondary test df", tags=["x"])
+    dataframe = DataframeMixin.log_dataframe(
```

Comment:
can you test the same for the `ArtifactMixin` in the tests at the beginning of the file?